sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathPermissions.java

```diff
@@ -278,19 +278,19 @@ public PathPermissions other(RolePermissions other) {
      *
      * @return {@code true} if the sticky bit is set and {@code false} otherwise.
      */
-    public boolean stickyBit() {
+    public boolean hasStickyBit() {
         return stickyBit;
     }
 
     /**
      * Sets the value of the sticky bit. The sticky bit may be set on directories, the files in that
      * directory may only be renamed or deleted by the file's owner, the directory's owner, or the root user.
      *
-     * @param stickyBit {@code True} to set the sticky bit and {@code false} to clear it.
+     * @param setStickyBit {@code True} to set the sticky bit and {@code false} to clear it.
      * @return The updated PathPermissions object.
      */
-    public PathPermissions stickyBit(boolean stickyBit) {
```

Comment:
is the argument supposed to be renamed like this?

Reply:
Sas is formatted like this 
/**
     * @return the read permission status
     */
    public boolean hasReadPermission() {
        return readPermission;
    }

    /**
     * Sets the read permission status.
     *
     * @param hasReadPermission Permission status to set
     * @return the updated BlobContainerSasPermission object
     */
    public BlobContainerSasPermission setReadPermission(boolean hasReadPermission) {
        this.readPermission = hasReadPermission;
        return this;
    }

Reply:
I think I should rename it to has